client/packages/system/src/Stock/api/hooks/useStockLine.ts

```diff
@@ -81,8 +81,9 @@ export function useStockLine(id?: string) {
     return result;
   };
   const update = async () => {
-    updateMutation(patch);
     setIsDirty(false);
+    await updateMutation(patch);
+    setPatch({});
```

Comment:
Unrelated, but something I noticed when testing. Best to clear the patch after update, so you start fresh with subsequent updates. (This was why the original "bug" didn't happen sometimes -- if you hadn't done something to clear the patch object, the original location would get updated, so no bug -- perhaps that's why you saw it "intermittently", @lache-melvin ?)